content/applications/websites/livechat/chatbots.rst

```diff
@@ -99,7 +100,7 @@ Email
 This step prompts visitors to provide their email address, which is stored and can be used by team
 members later to follow up with additional information.
 
-The only accepted inputs for this step type are email addresses that are in a valid format. If a
+The **only** accepted inputs for this step type are email addresses that are in a valid format. If a
```

Comment:
Just adding this here since I can't add it in the rest of this email section. You might want to mention that the question should be related to this- maybe in an example block? I was confused for a second, the realized the question has to be "Can you provide me with your email address so I can reach you?'or something like that. 

I was also going to recommend an example be added for each of these Step Types to fully illustrate the possibilities. But that might be overkill- however, it could help someone as they are trying to think of how to format their chatbot. Not necessary- just an idea!

Reply:
I agree examples for each would be overkill, but I think since the image with this section also has the question and "invalid email" response, it would be overkill here as well. I will add a line about formatting the question to prompt the visitor to enter their email address